### PR TITLE
Removed outdated section of primary package.json and add jsdoc comments

### PR DESCRIPTION
**Pull request by amoweolubusayo**
**Description**
A clear and concise description of the features you're adding in this pull request.

**Additional context**
Add any other context about the problem you're solving.

**Metadata**
- Fixes #[Link to Issue]

### CODE REVIEW

File: packages/contracts/src/contract-defs.ts [hunk 1]
```diff
@@ -14,11 +15,13 @@ export const getContractDefinition = (name: string): any => {
 }
 
 export const getContractInterface = (name: string): ethers.utils.Interface => {
+  //returns the interface for a given contract by name.
   const definition = getContractDefinition(name)
   return new ethers.utils.Interface(definition.abi)
 }
 
 export const getContractFactory = (
+  //returns the interface,bytecode and signer of a given contract by name.
```
Comment (tynes) [inline packages/contracts/src/contract-defs.ts:24]: These are non standard jsdoc comments, usually they go above the function definition with a particular format

Comment (amoweolubusayo) [reply packages/contracts/src/contract-defs.ts:24]: Thank you so much! My bad. I would correct my PR.

Comment (smartcontracts) [reply packages/contracts/src/contract-defs.ts]: New comments look great! Thank you :-)

**Review by smartcontracts — COMMENTED** [review]
Looking good, great job! Two quick things to resolve:

1. Let's add a changeset for the contracts package so that your new jsdoc comments are published as part of the `@eth-optimism/contracts` package. You can see instructions for how to do this [here](https://github.com/ethereum-optimism/optimism/blob/develop/CONTRIBUTING.md#changesets).
2. Let's also combine all of these changes into one commit that uses the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) format. You can do this by running the command `git rebase --interactive` and following the instructions that appear there.

File: packages/contracts/src/contract-defs.ts (hunk 1 above)
Comment (amoweolubusayo) [reply packages/contracts/src/contract-defs.ts]: Got the new comments..would work on those! Thanks!

Comment (amoweolubusayo) [reply packages/contracts/src/contract-defs.ts]: Done. changeset added and rebase done.

File: packages/contracts/src/contract-defs.ts [hunk 2]
```diff
@@ -1,6 +1,11 @@
 import { ethers } from 'ethers'
-
-export const getContractDefinition = (name: string): any => {
+/** 
+ * Gets the contract's artifact
```
Comment (maurelian) [inline packages/contracts/src/contract-defs.ts:3]: It's a nit, but I think this should have a period after it for consistency. 

This is what was done in the example given in @smartcontracts review:
https://github.com/ethereum-optimism/optimism/blob/3525635a3b240a92434016d2129534892ad24124/packages/core-utils/src/optimism/alias.ts#L14-L19

Comment (amoweolubusayo) [reply packages/contracts/src/contract-defs.ts:3]: Got it!

Comment (amoweolubusayo) [reply packages/contracts/src/contract-defs.ts]: Thank you for pointing that out.

Comment (amoweolubusayo) [reply packages/contracts/src/contract-defs.ts]: There are merge conflicts.. Am I to resolve that?

Comment (maurelian) [reply packages/contracts/src/contract-defs.ts]: Yes please.

Comment (maurelian) [reply packages/contracts/src/contract-defs.ts]: To do that, I would: 
1. fix the conflicts in package.json
2. rerun `yarn`, which will overwrite the yarn.lock file with the correct contents.

Also don't forget to combine the commits per the [comment above](https://github.com/ethereum-optimism/optimism/pull/2303#pullrequestreview-906171765). :)

Comment (amoweolubusayo) [reply packages/contracts/src/contract-defs.ts]: Thanks! I have made the updates

Comment (amoweolubusayo) [reply packages/contracts/src/contract-defs.ts]: Checks are failing. What can I do?

**Comment by maurelian** [comment]
@amoweolubusayo sorry but it looks like you'll have to pull from master again, and then please also combine all of these changes into one commit that uses the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) format. You can do this by running the command `git rebase --interactive `and following the instructions that appear there.

**Pull request closed without merging**